tests/run.bats

```diff
@@ -12,6 +12,11 @@ setup_file() {
   export BUILDKITE_PLUGIN_DOCKER_COMPOSE_RUN_LABELS="false"
 }
 
+teardown() {
+  # some test failures may leave this file around
+  rm -f docker-compose.buildkite-1-override.yml
```

Comment:
Don't believe it will make much of a difference other than for consistency but noticed in `tests/v2/run.bats` is using a glob
```
rm -f docker-compose.buildkite*-override.yml
```